src/components/tl/ucp/tl_ucp_tag.h

```diff
@@ -39,16 +39,19 @@
 #define UCC_TL_UCP_SCOPE_BITS_OFFSET                                        \
     (UCC_TL_UCP_ID_BITS + UCC_TL_UCP_SENDER_BITS + UCC_TL_UCP_SCOPE_ID_BITS)
 
-#define UCC_TL_UCP_SCOPE_ID_BITS_OFFSET (UCC_TL_UCP_ID_BITS + UCC_TL_UCP_SENDER_BITS)
+#define UCC_TL_UCP_SCOPE_ID_BITS_OFFSET (UCC_TL_UCP_ID_BITS + \
+                                         UCC_TL_UCP_SENDER_BITS)
 #define UCC_TL_UCP_SENDER_BITS_OFFSET   (UCC_TL_UCP_ID_BITS)
 #define UCC_TL_UCP_ID_BITS_OFFSET       0
 
-#define UCC_TL_UCP_MAX_TAG       UCC_MASK(UCC_TL_UCP_TAG_BITS)
-#define UCC_TL_UCP_RESERVED_TAGS 8
-#define UCC_TL_UCP_MAX_COLL_TAG  (UCC_TL_UCP_MAX_TAG - UCC_TL_UCP_RESERVED_TAGS)
-#define UCC_TL_UCP_SERVICE_TAG   (UCC_TL_UCP_MAX_COLL_TAG + 1)
-#define UCC_TL_UCP_MAX_SENDER    UCC_MASK(UCC_TL_UCP_SENDER_BITS)
-#define UCC_TL_UCP_MAX_ID        UCC_MASK(UCC_TL_UCP_ID_BITS)
+#define UCC_TL_UCP_MAX_TAG           UCC_MASK(UCC_TL_UCP_TAG_BITS)
+#define UCC_TL_UCP_RESERVED_TAGS     8
+#define UCC_TL_UCP_MAX_COLL_TAG     (UCC_TL_UCP_MAX_TAG - \
+                                     UCC_TL_UCP_RESERVED_TAGS)
```

Comment:
This is a code formatting only change and should be a separate patch (or dropped).